pkg/node/node_test.go

```diff
@@ -866,6 +866,17 @@ var _ = ginkgo.Describe("CSINodeService", func() {
 					}, nil)
 
 				arrays := getTestArrays()
+				clientMock.On("GetNASServers", mock.Anything).
+					Return([]gopowerstore.NAS{
+						{
+							NfsServers: []gopowerstore.NFSServerInstance{
+								{
+									IsNFSv4Enabled: true,
+									IsNFSv3Enabled: false,
+								},
+							},
+						},
+					}, nil)
```

Comment:
This return object seems to be the same across all tests, so you could just define a variable and use it in the tests.

Reply:
Updated